website/docs/workloads/redis/redis-profiles.md

```diff
@@ -50,11 +50,14 @@ idea. The name of the client must match the name of the system or the value of t
 ## PERF-REDIS.json
 Runs the Memtier workload against to generate various network traffic patterns against a Redis server. Although this is the default client workload
 the Redis benchmark itself can also be run to evaluate the performance of the Redis server.
+We have two profiles for Redis.One supports redis with TLS and one without TLS.
+
+* [Workload Profile without TLS](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-REDIS.json) 
+* [Workload Profile with TLS](https://github.com/microsoft/VirtualClient/blob/main/src/VirtualClient/VirtualClient.Main/profiles/PERF-REDIS-TLS.json) 
```

Comment:
Do we have any doc on how to generate the .crt or refresh it? Does the crt has an expiration?

Reply:
We have a script file In Redis that generates these certificates and keys. It's part of redis binaries. As of I know we don't have expiration for these crt and keys. Rather than generating crt and keys everytime we run the workload, we can generate them once and can use those. 
In Redis folder we have the script file at redis/utils/gen-test-certs.sh which generates crt and keys at redis/tests/tls folder.

Reply:
So by putting those keys in source, does that mean anyone can get the key we generate and decrypt the redis messages? Which might be fine because this is benchmarking with fake data, but we might want to make it clear in the document

Reply:
Correct. Here we are not concerned about the security aspect but just the benchmarking results. Sure will update the document to make it more clear. Thanks Yang